### PR TITLE
Mining Bluespace Survival Capsule's "Stand back!" alert now actually has meaning 

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

- Standing in range of a Bluespace Survival Capsule being deployed will now toss you away from the deployed location of the pod, to roughly 2 tiles away.

## Why It's Good For The Game

I always found it weird (and a little misleading) that triggering a survival capsule alerts people to "Stand back!", but it's actually harmless to stand on top of it and even smart in some contexts since you phase into it like nothing happened. 

So I added the missing flavor. 

This also has some potential shenanigans involving antagging, as you can time it perfectly so that you toss it just in time to throw people away from it.

Note: The Luxury Elite pod is so large, that even after being thrown away, you may still end up inside the pod by the time it deploys, but I think that's funny. So. 

## Changelog

:cl: Melbert
add: Shaft Miner's Bluespace Survival Capsules will now throw people away from it when it deploys. Be sure to heed the warning to "Stand back". 
/:cl:

### CODE REVIEW

File: code/modules/mining/equipment/survival_pod.dm [hunk 1]
```diff
@@ -37,35 +37,77 @@
...
+		if(SHELTER_DEPLOY_BAD_TURFS, SHELTER_DEPLOY_ANCHORED_OBJECTS, SHELTER_DEPLOY_OUTSIDE_MAP)
+			loc.visible_message(span_warning("[src] doesn't have room to deploy! You need to clear a [width]x[height] area!"))
+
+	if(status != SHELTER_DEPLOY_ALLOWED)
+		used = FALSE
+		return
+
+	var/base_x_throw_distance = ceil(width / 2)
+	var/base_y_throw_distance = ceil(height / 2)
+	for(var/mob/living/did_not_stand_back in range(loc, "[width]x[height]"))
```
Comment (san7890) [inline code/modules/mining/equipment/survival_pod.dm:75]: i feel like this whole chunk would be better off in another prompt for clarity purposes (e.g. this proc can be easily read for template spawning/handling, and any behavior relating to "throwing mobs" can be read in the separate proc)

**Review by san7890 — CHANGES_REQUESTED** [review]
nit

**Comment by the-orange-cow** [comment]
If possible, maybe the bar capsule would not behave like this?  It is a high price item, so maybe they added special safety features to it?

**Comment by Admiral-Gigaton** [comment]
> If possible, maybe the bar capsule would not behave like this? It is a high price item, so maybe they added special safety features to it?

yeah, instead it just breaks your leg, but has complementary bone repair meds in the bathroom (joke)

**Comment by atlasle** [comment]
> If possible, maybe the bar capsule would not behave like this?  It is a high price item, so maybe they added special safety features to it?

no it should be stronger and toss you even farther since it's a larger building, respect safety measures!

**Comment by MrMelbert** [comment]
> If possible, maybe the bar capsule would not behave like this? It is a high price item, so maybe they added special safety features to it?

Well I mentioned it in the PR body but the bar capsule is so large that sometimes, despite throwing you, you will get caught by its walls, so we can call that a feature

**Comment by Exester509** [comment]
> If possible, maybe the bar capsule would not behave like this? It is a high price item, so maybe they added special safety features to it?

Money can't get you away from OSHA

**Comment by projectkepler-ru** [comment]
You should make it have the Trauma Team 'Stand Clear' holoprojector effect like in cyberpunk

**Review by ArcaneMusic — CHANGES_REQUESTED** [review]
minor nits

**Review by Ghommie — APPROVED** [review]

**Pull request merged**